test_checkIncoming  fails when a direction on my system is not empty

Looks to me like this test still somehow points to a LANL/NMC directory. This folder exists and those are its files on my system (spacesci-1).

All other tests pass
 

```
FAIL: test_checkIncoming (test_DButils.DBUtilsOtherTests)
checkIncoming
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ectsoc/git/dbprocessing/unit_tests/test_DButils.py", line 123, in test_checkIncoming
    self.assertFalse(self.dbu.checkIncoming())
AssertionError: [u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_34a_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_34a_02.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_35a_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_35a_02.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_36a_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_36a_02.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_375_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_375_02.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_382_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0009_382_02.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0019_375_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0019_382_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0025_37e_04.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0025_37f_05.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0025_380_05.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0025_381_05.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0026_37f_01.ptp.gz', u'/n/space_data/cda/rbsp/mageis_incoming/ect_rbspa_0026_380_01.ptp.gz', u'/n/sp
<snip>
```

there are a lot of files in there

--- a/unit_tests/test_DButils.py
+++ b/unit_tests/test_DButils.py
@@ -120,14 +120,19 @@ class DBUtilsOtherTests(TestSetup):
 
     def test_checkIncoming(self):
         """checkIncoming"""
-        self.assertFalse(self.dbu.checkIncoming())
         e = self.dbu.getEntry('Mission', 1)
-        e.incoming_dir = os.path.abspath(os.path.dirname(__file__))
-        self.dbu.session.add(e)
-        self.dbu.commitDB()
-        inc_files = self.dbu.checkIncoming()
-        self.assertTrue(inc_files)
-        self.assertTrue(os.path.join(os.path.abspath(os.path.dirname(__file__)), 'test_DButils.py') in inc_files)
+        td = tempfile.mkdtemp()
+        try:
+            e.incoming_dir = td
+            self.assertFalse(self.dbu.checkIncoming())
+            fnames = [os.path.join(td, f) for f in ['test', 'test2', '3test']]
+            for f in fnames:
+                open(f, 'w').close()
+            inc_files = self.dbu.checkIncoming()
+            self.assertTrue(inc_files)
+            self.assertEqual(sorted(fnames), sorted(inc_files))
+        finally:
+            shutil.rmtree(td)
 
     def test_currentlyProcessing(self):
         """currentlyProcessing"""

